HdM-LLM/backend
# added an example of a vector store with generated cover letters from chatGPT

If you want to test it before merging, follow these steps:

1. comment in lines 95-99 (lines with `load_document()`) in `vector_store.py`
2. comment out the `search_similar_db()`function  in lin 102 ff.
3. run the script
4. comment out lines 95-99 again, so the cover letters do not get added again
5. comment in the `search_similar_db()` function  in lin 102 ff. 
6. run the script

Your console output should look like this:
![image](https://github.com/HdM-LLM/backend/assets/95630400/958ce0c1-91de-4312-8f11-66f5a6803b72)
